core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java

```diff
@@ -2452,7 +2453,7 @@ private Scope createScopeForView(Table table, QualifiedObjectName name, Optional
                     view.getRunAsIdentity(),
                     view.getPath(),
                     view.getColumns(),
-                    Optional.empty());
+                    Optional.empty(), false);
```

Comment:
nit: Can we move `false` to a new line ?

Reply:
oops!